src/main/java/com/k2view/agent/postman/CloudManager.java

```diff
@@ -35,25 +36,50 @@ public CloudManager(String mailboxId, String mailboxUrl) {
         }
         this.mailboxId = mailboxId;
         this.uri = URI.create(mailboxUrl);
-        this.client = HttpClient.newBuilder().build();
+
```

Comment:
add env params to configure the different cases

Reply:
CloudManager.java is your code. Implement my code in your code however you want